postgres-conn/pgpass-privatenet

```diff
@@ -1 +1 @@
-/var/run/postgresql:5432:privatenet:*:*
+localhost:5432:privatenet:*:*
```

Comment:
This makes me realize that the PostgreSQL set up needs some love.  Perhaps, we need to put the pgpass file into the templates folder and then use the `db-sync-start.sh` script to copy it inside the private-testnet folder and use sed to substitute the host based on user variable.  By default, it appears a Linux install (at least for Ubuntu) uses local unix sockets instead of TCP/IP listener connections.  And maybe Homebrew does it the other way by default?

The create database instructions (Step 4 of the 3-RUN-SCRIPTS.md) could be scripted into the `db-sync-start.sh` file as well.  The script could automatically do a drop and create of the database, so that you always start with a clean database, whenever a new private-testnet is paved.

What do you think?

Reply:
Yeah, this seems like something that should be templated based on platform, or even a setting that allows someone to specify how they want their pgpass to look.